sdk/core/core-http/src/policies/tracingPolicy.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 import { getTracer, getTraceParentHeader } from "@azure/core-tracing";
-import { SpanOptions, SpanKind } from "@opentelemetry/types";
+import { SpanOptions, SpanKind } from "@opentelemetry/api";
```

Comment:
`SpanOptions` in app config is being imported from `@azure/core-tracing`, but here we are importing from `@opentelemetry/api`?

Reply:
This is because the `SpanOptions` we're building in the policy is for creating a `Span`, not providing public surface for consumers. In other places I renamed the import to make this clear (`OTSpanOptions`) while this isn't required for this usage, should I follow the same pattern to avoid confusion?